.claude/commands/debug.md

```
@@ -48,6 +72,81 @@ Protocol Phases:
+
+Output (--assess):
+
+```
+Complexity Assessment
+------
+Bug: Distributed timeout across payment and notification services
+
+Score: 0.72 / 1.0
+Suggested Agents: 3
+Factors:
+  - multi_service (0.67): Detected: distributed, microservice
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)